### ISSUE TITLE
C:\\* returns empty list

### ISSUE THREAD

**Issue by gbaumgart** [isaacs/node-glob#290]
hi,
if you pass `C:\\*` or `C:/*` its returning nothing. Is there are trick to get the folder/file list on a root disc ? 
thanks,
g

**Comment by phated** [comment]
`root` is already an option that is resolved to `C:\\` on Windows.  You are supposed to use globs that look like unix paths.  So you'd use `/` always (even on Windows).

**Comment by gbaumgart** [comment]
yeah, realized this. sorry & thanks

**Comment by isaacs** [comment]
I'd still call this a bug.  `c:\\*` should be functionally equivalent to `/*` on Windows.

**Comment by gbaumgart** [comment]
yup, it is because currently i am forced to set root and add it to the result which can be quite error-prone

**Comment by nigurr** [comment]
This also fails if I am giving pattern like

`C:\\Test\\Test.txt` and some `C:\\Test\\Test*.txt` and I am executing from different drive.

I debugged and found that https://github.com/isaacs/node-glob/blob/master/sync.js#L95 this looks problematic. It passes wrong `prefix` and there by `searching on the wrong folder`
